spec/models/occupation_standard_spec.rb

```diff
@@ -510,4 +510,30 @@
       expect(occupation_standard.display_for_typeahead).to eq "Mechanic (15-1342) (9201)"
     end
   end
+
+  describe "#hours_meet_occupation_requirements?" do
+    it "returns true if work_process hours match occupation hours" do
+      occupation = create(:occupation, time_based_hours: "2000")
```

Comment:
Since your method is comparing the `work_process` hours only, let's make `time_based_hours` a different value (perhaps less than 2000), so the test is clear on what it is comparing against.

Reply:
Also, the hours should technically be an integer. Rails will adjust it, but it will avoid confusion if you put it in as the expected data type: `time_based_hours: 2000`

Reply:
I'll fix the time_based hours. ojt hours were a string so I accidently carried that over to occupation through copy and past.